properties/P000025.md

```diff
@@ -1,11 +1,16 @@
 ---
 uid: P000025
-slug: sigma-locally-compact
-name: "$\\sigma$-Locally Compact"
+slug: exhaustible-by-compacts
+name: Exhaustable by compacts
+aliases:
+  - "$\\sigma$-Locally Compact"
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
```

Comment:
Could we also add https://en.wikipedia.org/wiki/Exhaustion_by_compact_subsets as a reference, to better justify the name?